server/trillian_log_server_test.go

```diff
@@ -182,7 +182,7 @@ func TestGetLeavesByIndexMultiple(t *testing.T) {
 func TestQueueLeavesStorageError(t *testing.T) {
 	test := newParameterizedTest("QueueLeaves",
 		func(t *storage.MockLogTX) {
-			t.On("QueueLeaves", []trillian.LogLeaf{leaf0}).Return(errors.New("STORAGE"))
+			t.On("QueueLeaves", []trillian.LogLeaf{leaf1}).Return(errors.New("STORAGE"))
```

Comment:
did you change from leaf0 to leaf1 because leaf0 has no SequenceNumber? does it need to stay that way?